docs/sources/developers/http_api/user.md

```diff
@@ -24,11 +24,15 @@ refs:
 
 # User API
 
-The Users HTTP API does not currently work with an API Token. API Tokens are linked to an organization and an organization role. They cannot be given
-the permission of server users access, only users can be given that permission. To use these API calls you can use Basic Auth and the Grafana
-user must have the Grafana Admin role.
+{{% admonition type="caution" %}}
+The Users HTTP API currently does not work with service account tokens. Service accounts are limited to an organization and an organization role. They cannot be granted [Grafana server administrator permissions]({{< relref "../roles-and-permissions/#grafana-server-administrators" >}}).
+In order to use these API endpoints you have to use Basic Auth and the Grafana user must have the Grafana server administrator permission.
 
-API Tokens can be used with Organization HTTP API to get users of specific organization.
+The `admin` user that Grafana is provisioned with by default has permissions to use these API endpoints.
+
+Service account tokens can be used with Organization HTTP API to get users of specific organization.
```

Comment:
Should this be part of the caution in `org.md` instead?

Reply:
Oh I see now, that makes sense

Reply:
Maybe something like "Alternatively, you can use the Organization HTTP API with service account tokens to get the users for a specific organization"

Reply:
Just changed it to your suggestion, let me know if it looks good to you now. And thanks a lot for making this much neater and providing so much context on your suggestions!